test/infrastructure/inmemory/pkg/server/api/const.go

```diff
@@ -142,6 +142,25 @@ var (
 				},
 				StorageVersionHash: "",
 			},
+			{
+				Name:         "namespaces",
+				SingularName: "namespace",
+				Namespaced:   false,
+				Kind:         "Namespace",
+				Verbs: []string{
+					"create",
+					"delete",
```

Comment:
deletecollection doesn't exist for namespace right?

Reply:
no (it is copy paste from the output of a /api/v1 discovery call of a real API server)